app/assets/javascripts/mobile_tooltip.js

```diff
@@ -0,0 +1,41 @@
+(function () {
+  /*
+  **  Tooltip for the profile page on mobile devices
+  **  This script should ideally run on the profile page
+
+  **  Behaviour
+      ========================
+      - Enables the viewers tooltip to be shown on mobile devices when the
+        parent element is clicked.
+      - Toggle behaviour is implemented so that the tooltip can be both shown
+        and dismissed
+
+  */
+
+  $(document).on("page:load ready", function () {
+    // Allow this script to only be run on the profile page
+
+    if (window.location.pathname !== "/profile") return false;
```

Comment:
I don't think this is strictly necessary... `some_focus_text` is only used in the profile page, and if we ever have another use case for mobile tooltips on a different page, we should just keep using that class (or add a new one). I think this falls enough into a "general utility" category that we don't have to restrict its usage with an early return like this.